src/workflowOptions.ts

```diff
@@ -58,6 +68,32 @@ export const workflowOptions = {
     ],
     initial: 0,
   },
+  signingOption: {
```

Comment:
Lets simply not show this option, if the previous selected template is not the TG app.
For reference https://github.com/terkelg/prompts?tab=readme-ov-file#dynamic-prompts

```
  {
    type: prev => prev.path == "clicker-game-tg-mini-app-template" ? 'selection' : null,
    name: 'signingOption',
    message: 'Choose your signing option',
    choices:[
       {
            title: "Explicit signing (Aptos Wallet Adapter)",
            value: "explicit",
          },
          {
            title: "Seamless signing (Mizu SDK Core)",
            value: "seamless",
          }
    ]
  }
```